src/config/quantTypes.wip.json

```diff
@@ -1307,7 +1307,7 @@
       "units": "nM",
       "assay": {
         "type": "Heron TapeStation Tubes",
-        "version": "v1.0"
+        "version": "v2.0"
```

Comment:
Just checking. Do we actually use this file at all anymore?

Reply:
Not as far as I'm aware.  It adds `heronQPCR` as a quant type, but it doesn't look like it was ever released.  I wasn't happy to just delete it when I don't understand why it wasn't released yet, but I guess it's in the git history now so I should just delete it and we'll dig it up if needed.

Reply:
Ah yes. This was a piece of work that I halted because it turned out to be complex with little value. We can bin it.